creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -123,31 +178,59 @@ const updateReplicaSetJobProcessor = async function ({
       replicaToUserInfoMap,
       enabledReconfigModes
     })
-    ;({ errorMsg, issuedReconfig, syncJobsToEnqueue } =
-      await _issueUpdateReplicaSetOp(
-        userId,
-        wallet,
-        primary,
-        secondary1,
-        secondary2,
-        newReplicaSet,
-        audiusLibs,
-        logger
-      ))
+
+    try {
```

Comment:
i thought we discussed unnesting this?

Reply:
also `issueUpdateReplicaSetOp()` never throws so this try-catch just seems unnecessary?

Reply:
I thought getting rid of try-catch was something you wanted in a future refactor

Also, we don't ever explicitly throw an error in that function, but if we do, it'll get caught appropriately with that try-catch block

Reply:
I had it unnested with the custom `Error` types but renested them because unnesting them would mess up the control flow of the function.

Reply:
sure lets just ship, i think still works